README.md

```diff
@@ -120,14 +120,22 @@ This example will dump all replication events to the console:
 ```python
 from pymysqlreplication import BinLogStreamReader
 
-mysql_settings = {'host': '127.0.0.1', 'port': 3306, 'user': 'root', 'passwd': ''}
+MYSQL_SETTINGS = {"host": "127.0.0.1", "port": 3306, "user": "root", "passwd": ""}
 
-stream = BinLogStreamReader(connection_settings = mysql_settings, server_id=100)
 
-for binlogevent in stream:
-    binlogevent.dump()
+def main():
+    stream = BinLogStreamReader(
+    connection_settings=MYSQL_SETTINGS, server_id=3, blocking=True
+    )
 
-stream.close()
+    for binlogevent in stream:
+        binlogevent.dump()
+
+    stream.close()
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
I didn't mean for you to change this area. 

I was asking to change the link on line 115.
I'm sorry I didn't convey my intentions clearly.

Reply:
Sorry It was my mistake. I resolved the issue now. Changed the "Example Directory" 
from https://github.com/noplay/python-mysql-replication/tree/master/examples 
to https://github.com/noplay/python-mysql-replication/tree/main/examples

additionally I roll backed the changes done to Example and Output code in example block.